src/ducks/client/links.js

```diff
@@ -1,4 +1,4 @@
-/* global __TARGET__ */
+/* global __TARGET__, __DEV__ */
```

Comment:
If I'm not mistaken, the global is `__DEVELOPMENT__`, as seen in https://github.com/cozy/cozy-banks/blob/master/config/webpack.config.dev.js#L26